### PR TITLE
#134 Improved syntax highlighting

### PR DESCRIPTION
**Pull request by StephanHagge**
Fixed error string highlighting on multiline messages.
Prevent highlighting of "keywords" inside other words.

### CODE REVIEW

File: frontend/src/app/ruleset-editor/ruleset-editor.component.ts [hunk 1]
```diff
@@ -360,7 +360,7 @@ export class RulesetEditorComponent implements OnInit, OnDestroy {
             if (value.pattern === 'string.unquoted.ov') {
               const thenInRange = this.editor.getModel().getValueInRange({
                 startLineNumber: value.range.start.line + 1,
-                endLineNumber: value.range.end.line + 1,
+                endLineNumber: value.range.start.line + 1,
```
Comment (Jaykju) [inline frontend/src/app/ruleset-editor/ruleset-editor.component.ts:363]: Why was this changed? Are there no multiline strings?

Comment (StephanHagge) [reply frontend/src/app/ruleset-editor/ruleset-editor.component.ts:363]: This is a fix. At this point we try to define the range in which a "then" could appear before a string. This range begins with the first character, which is in the line in which the string begins. It ends with the last character before the string. This means that only the line in which the string begins is checked.
Previously, the range instead started with the first character in the line of the string starts, and ended before the position (column) of the first character, but in the line of the string end. This was not a problem with single-line strings, since the lines between the start and end of the string were the same. However, an incorrect range was created for multi-line strings.
